### PR TITLE
[docs] cleanup project documentation

### PR DESCRIPTION
**Pull request by dariuszkuc**
Cleaned up README and added new CONTRIBUTING doc (that also includes how to release this project).

Resolves: https://github.com/apollographql/federation-jvm/issues/178

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Contributing to Federation JVM
+
+The Apollo team welcomes contributions of all kinds, including bug reports, documentation, test cases, bug fixes, and features. There are just a few guidelines you need to follow which are described in detail below.
+
+If you want to discuss the project or just say hi, stop by [Apollo community forums](https://community.apollographql.com/).
```
Comment (martinbonnin) [inline CONTRIBUTING.md:5]:
```suggestion
If you want to discuss the project or just say hi, stop by [the Apollo community forums](https://community.apollographql.com/).
```
```suggestion
If you want to discuss the project or just say hi, stop by [Apollo community forums](https://community.apollographql.com/).
```

Comment (martinbonnin) [inline CONTRIBUTING.md:5]:
```suggestion
If you want to discuss the project or just say hi, stop by [the Apollo community forums](https://community.apollographql.com/).
```
```suggestion
If you want to discuss the project or just say hi, stop by [Apollo community forums](https://community.apollographql.com/).
```

**Review by martinbonnin — APPROVED** [review]
👍

File: README.md [hunk 2]
```diff
@@ -5,82 +5,174 @@
...
+```mermaid
+graph BT;
+  gateway([Supergraph<br/>gateway]);
+  serviceA[Users<br/>subgraph];
+  serviceB[Products<br/>subgraph];
+  serviceC[Reviews<br/>subgraph];
+  gateway --- serviceA & serviceB & serviceC;
+```
+
+`graphql-java-support` is built on top of `graphql-java` library and provides transformation logic to make your GraphQL schemas Federation compatible. `SchemaTransformer` adds common Federation
```
Comment (BoD) [inline README.md:21]: Nitpick:
```suggestion
`graphql-java-support` is built on top of the `graphql-java` library and provides transformation logic to make your GraphQL schemas Federation compatible. `SchemaTransformer` adds common Federation
```

or 
```suggestion
`graphql-java-support` is built on top of `graphql-java` and provides transformation logic to make your GraphQL schemas Federation compatible. `SchemaTransformer` adds common Federation
```

**Review by BoD — APPROVED** [review]

**Pull request merged**